### DISCUSSION TITLE
Board not showing up after updating bootloader

### DISCUSSION THREAD

**Discussion by SnappierSoap318** [adafruit/Adafruit_nRF52_Bootloader#335] [Q&A]
Hi, 

I was flashing my Supermini nrf52840, which was running 0.6.0 bootloader to the latest version.

The UF2 bootloader didn't work, I guess, because the uf2 info file didn't update to show the latest version.

I tried flashing the bootloader using CLI and I got the `Device Programmed` message. 

After that the board doesn't seem to boot or is probably in a boot loop.  

The inbuilt blinky doesn't load and on trying to put the board into DFU mode doesn't seem to work either. 

any insights on how i should proceed?

**Comment by dhalbert** [comment]
Which version are you using? See also this new issue: #334.

**Reply by SnappierSoap318** [reply]
my issue is similar to that but i wasnt able to downgrade the bootloader.

The board was running `0.6.0`, I tried flashing `0.9.1`.
